WordPress/src/main/java/org/wordpress/android/viewmodel/quickstart/QuickStartViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.viewmodel.quickstart
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import org.wordpress.android.fluxc.store.QuickStartStore
+import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTask
+import org.wordpress.android.ui.quickstart.QuickStartTaskState
+import javax.inject.Inject
+
+class QuickStartViewModel @Inject constructor(private val quickStartStore: QuickStartStore) : ViewModel() {
+    private val _quickStartTaskStateStates = MutableLiveData<List<QuickStartTaskState>>()
+    val quickStartTaskStateStates: LiveData<List<QuickStartTaskState>>
+        get() = _quickStartTaskStateStates
```

Comment:
NIT: you could do this instead `val quickStartTaskStateStates: LiveData<List<QuickStartTaskState>> = _quickStartTaskStateStates